core/test/unit/metadata/image-size_spec.js

```diff
@@ -0,0 +1,123 @@
+var should         = require('should'),
+    sinon          = require('sinon'),
+    rewire         = require('rewire'),
+
+// Stuff we are testing
+    getImageDimensions = rewire('../../../server/data/meta/image-dimensions'),
+
+    sandbox = sinon.sandbox.create();
+
+describe('getImageDimensions', function () {
+    var firstSizeOfStub, secondSizeOfStub, thirdSizeOfStub;
+
+    beforeEach(function () {
+        firstSizeOfStub = sandbox.stub();
```

Comment:
@kirrg001: Maybe you have a tip for me? I couldn't manage here to change my stub for each `it`-block, that's why I used three different variables, although it's pretty ugly. I tried as well, declare the stubs in `beforeEach` functions in each `it`-block, but that didn't work either. The only way it was working, was when I tested each statement separately, but that's not the solution.
Happy for feedback!